runtime/devnet/src/extensions.rs

```diff
@@ -346,544 +394,501 @@ where
-		new_test_ext().execute_with(|| {
-			let _ = env_logger::try_init();
-
-			let (wasm_binary, _) = load_wasm_module::<Runtime>(
-				"../../pop-api/examples/place-spot-order/target/ink/spot_order.wasm",
-			)
-			.unwrap();
-
-			let init_value = 100 * UNIT;
-
-			let result = Contracts::bare_instantiate(
-				ALICE,
-				init_value,
-				GAS_LIMIT,
-				None,
-				Code::Upload(wasm_binary),
-				function_selector("new"),
-				vec![],
-				DEBUG_OUTPUT,
-				pallet_contracts::CollectEvents::Skip,
-			)
-			.result
-			.unwrap();
-
-			assert!(!result.result.did_revert(), "deploying contract reverted {:?}", result);
-
-			let addr = result.account_id;
-
-			let function = function_selector("place_spot_order");
-
-			let max_amount = 1 * UNIT;
-			let para_id = 2000;
-
-			let params = [function, max_amount.encode(), para_id.encode()].concat();
-
-			let result = Contracts::bare_call(
-				ALICE,
-				addr.clone(),
-				0,
-				Weight::from_parts(100_000_000_000, 3 * 1024 * 1024),
-				None,
-				params,
-				DEBUG_OUTPUT,
-				pallet_contracts::CollectEvents::Skip,
-				pallet_contracts::Determinism::Enforced,
-			);
-
-			if DEBUG_OUTPUT == pallet_contracts::DebugInfo::UnsafeDebug {
-				log::debug!(
-					"Contract debug buffer - {:?}",
-					String::from_utf8(result.debug_message.clone())
-				);
-				log::debug!("result: {:?}", result);
-			}
-
-			// check for revert
-			assert!(!result.result.unwrap().did_revert(), "Contract reverted!");
-		});
-	}
-
-	#[test]
-	#[ignore]
-	fn allow_call_filter_blocks_call() {
-		new_test_ext().execute_with(|| {
-			let _ = env_logger::try_init();
-
-			let (wasm_binary, _) = load_wasm_module::<Runtime>(
-				"../../tests/contracts/filtered-call/target/ink/pop_api_filtered_call.wasm",
-			)
-			.unwrap();
-
-			let init_value = 100 * UNIT;
-
-			let result = Contracts::bare_instantiate(
-				ALICE,
-				init_value,
-				GAS_LIMIT,
-				None,
-				Code::Upload(wasm_binary),
-				function_selector("new"),
-				vec![],
-				DEBUG_OUTPUT,
-				pallet_contracts::CollectEvents::Skip,
-			)
-			.result
-			.unwrap();
-
-			assert!(!result.result.did_revert(), "deploying contract reverted {:?}", result);
-
-			let addr = result.account_id;
-
-			let function = function_selector("get_filtered");
-			let params = [function].concat();
-
-			let result = Contracts::bare_call(
-				ALICE,
-				addr.clone(),
-				0,
-				Weight::from_parts(100_000_000_000, 3 * 1024 * 1024),
-				None,
-				params,
-				DEBUG_OUTPUT,
-				pallet_contracts::CollectEvents::Skip,
-				pallet_contracts::Determinism::Enforced,
-			);
-
-			if DEBUG_OUTPUT == pallet_contracts::DebugInfo::UnsafeDebug {
-				log::debug!(
-					"Contract debug buffer - {:?}",
-					String::from_utf8(result.debug_message.clone())
-				);
-				log::debug!("filtered result: {:?}", result);
-			}
-
-			// check for revert
-			assert!(!result.result.unwrap().did_revert(), "Contract reverted!");
-		});
-	}
-}
+// use enumflags2::BitFlags;
```

Comment:
A reminder to resolve.

Reply:
These should be done in a separate PR after this one gets merged. It will require quite some refactoring based on the new design.